### PR TITLE
chore: upgrade wrangler and ci

### PR DESCRIPTION
**Pull request by markusahlstrand**

### CODE REVIEW

File: yarn.lock
Comment (danielgent) [inline yarn.lock:1]: what have you updated here without changing the package.json?

i'll make the same comment I always make, that we should use renovate bot so it does a single minimum upgrade per PR

I remember you saying _It doesn't do much it only takes a second to update the dependencies_ but it doesn't take a second to fix the failing CI/CD - finding the specific package and specific version that causes the issue is like finding a needle in a haystack

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
         run: yarn semantic-release
       - run: yarn build
       - name: Deploy
-        uses: cloudflare/wrangler-action@2.0.0
+        uses: cloudflare/wrangler-action@v3
```
Comment (danielgent) [inline .github/workflows/deploy-dev.yml:30]: these should be pins else they'll upgrade whenever.   they should even be pinned to the hash (as renovate bot does)

Comment (markusahlstrand) [reply .github/workflows/deploy-dev.yml:30]: I'm not sure how it works.. I copied this from this page: https://github.com/cloudflare/wrangler-action

File: yarn.lock
Comment (markusahlstrand) [reply yarn.lock:1]: I tried to upgrade both the ci and wrangler but the build failed.. So I tried rolling back wrangler, but it still seems to fail.  Really odd.

**Comment by danielgent** [comment]
Closing this as I'm doing manual upgrades with an upgrade per PR, then letting the CI/CD give me :heavy_check_mark:

**Pull request closed without merging**